src/java/org/tensorics/core/tensor/Position.java

```diff
@@ -83,16 +83,16 @@ public static Position of(Object... coordinates) {
 		return createFrom(requireValidCoordinates(ImmutableMultiset.copyOf(coordinates)));
 	}
 
-	public ClassToInstanceMap<Object> getCoordinates() {
-		return coordinates;
+	public Set<Object> getCoordinates() {
+		return new HashSet<>(coordinates);
 	}
```

Comment:
Do we really need getCoordinates() and coordinates(), as they now behave identically? I see the point of not breaking the API, but I think then we should deprecate one and remove it later...